### DISCUSSION TITLE
PowerApp Migration error when migrating images folder within Assets

### DISCUSSION THREAD

**Discussion by lordneeko** [Zerg00s/FlowPowerAppsMigrator#59] [Q&A] [answered]
I wanted to start the conversation here first before submitting an issue.  The migrator worked perfectly moving my lists from one site to another, however when it starts processing the PowerApp zip package, something fails in handling the image files within the src zip.
See the error below.
Please help me figure out what is up here. Everything else appears to be working well!

![image](https://github.com/Zerg00s/FlowPowerAppsMigrator/assets/11577533/806facf9-4f50-484d-96fa-e7477a17b792)

**Comment by lordneeko** [comment]
I may have figured it out..... There may be a copy operation that is trying to happen in the background (that isn't shown in the output window) that is failing due to the path being too long.  If this is the case, then that error message needs to someone be shown to the user
![image](https://github.com/Zerg00s/FlowPowerAppsMigrator/assets/11577533/041d9332-1ce9-4769-9109-d31778e5ee52)

**Comment by lordneeko** [comment, marked as answer]
Yes, that was the solution.  I moved the Migrator folder to C:\Temp and it worked.

**Reply by lordneeko** [reply]
Suggest updated the Readme to advise the user to put the package as low on the root drive as possible.

**Reply by Zerg00s** [reply]
@lordneeko, thank you! I will update the readme with your recommendation.
